build/media-suite_compile.sh

```diff
@@ -599,7 +599,7 @@ fi
 unset _deps
 
 _check=(libopus.{,l}a opus.pc opus/opus.h)
-if enabled libopus && do_vcs "https://github.com/xiph/opus.git"; then
+if enabled libopus && do_vcs "https://git.xiph.org/opus.git"; then
```

Comment:
Split these git URL changes to a separate commit (for all of them to git.xiph.org).